### DISCUSSION TITLE
Latest patch release of [@types/babel__traverse] v7.18.4 is broken:

### DISCUSSION THREAD

**Discussion by kumaran-is** [DefinitelyTyped/DefinitelyTyped#65299] [Issues with a @types package]
# Problem/Issue Description:
 Our stencil.js component build started failing since yesterday due to the default installation of the latest version of `@types/babel__traverse `( ie. version 7.18.4) by the `"babel-jest": "~29.5.0"` library. It appears that the recently released patch version 7.18.4 is broken. 

# Our Environment setup

node.js v16
"typescript": "~4.9.0"
"babel-jest": "~29.5.0",
"@stencil/core": "~3.2.0"

# Temporary Solution: 
To resolve this issue, we explicitly added the previous patch version of @types/babel__traverse (`version 7.18.3`) to  package.json .i.e `"@types/babel__traverse": "7.18.3" `

[a comment by an automated account is not shown]

**Comment by danez** [comment]
There is already a revert in progress: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65298
